components/homepage/HomeScreen.js

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+import { List, Checkbox } from 'react-native-paper';
+import { SafeAreaView } from 'react-native-safe-area-context';
+import { useSelector, useDispatch } from 'react-redux';
+import { View, Text, Button, TouchableOpacity, ScrollView } from 'react-native';
+import { fetchAllTasks, updateTasks } from '../../redux/TasksSlice';
+import { fetchAllTmrTasks, updateTmrTasks } from '../../redux/TomorrowTasksSlice';
+import { Bars3CenterLeftIcon, PlusCircleIcon } from "react-native-heroicons/solid";
+import { colors } from '../../theme/colors';
+import { formatTime } from '../../helper/TimeFormat';
+
+const HomeScreen = ({ navigation }) => {
+
+    const dispatch = useDispatch();
+
+    const tasks = useSelector(state => state.tasks.listTasks);
```

Comment:
You can use reselect to create selector, this will help to memorized selectors https://redux.js.org/usage/deriving-data-selectors#createselector-overview

```
const tasksSelector = createSelector(state => state.tasks, tasks => {
  return tasks.listTasks
})

const tasks = useSelector(tasksSelector)
```